tsdoc/src/__tests__/__snapshots__/DocNodeTransforms.test.ts.snap

```diff
@@ -27,7 +41,28 @@ Object {
     },
     Object {
       "kind": "PlainText",
-      "nodePlainText": "sentence. This is another sentence.",
+      "nodePlainText": "sentence.",
+    },
+    Object {
+      "kind": "SoftBreak",
+    },
+    Object {
+      "kind": "PlainText",
+      "nodePlainText": " This is another",
+    },
+    Object {
+      "kind": "SoftBreak",
+    },
+    Object {
+      "kind": "PlainText",
+      "nodePlainText": " sentence.",
+    },
+    Object {
+      "kind": "SoftBreak",
+    },
+    Object {
+      "kind": "PlainText",
+      "nodePlainText": "",
```

Comment:
Trailing line break shouldn't be here (or in other updated tests). Need to fix this.